react/components/Share/components/SocialButton.js

```diff
@@ -43,6 +43,8 @@ export default class SocialButton extends Component {
     iconClass: PropTypes.string,
     /** Image url to share in social medias */
     imageUrl: PropTypes.string,
+    /** Function that sends share event to GA4 */
```

Comment:
Let's generalize this to "analytics" since it can be used by more than just GA4.

There's another comment just like this in another file.

```suggestion
    /** Function that sends share event to analytics */
```